src/server/gc_worker.rs

```diff
@@ -203,6 +207,80 @@ impl<E: Engine> GcRunner<E> {
         .map_err(Error::from)
     }
 
+    /// Check need gc without getting snapshot.
+    /// If this is not supported or any error happens, returns true to do further check after
```

Comment:
There will be a switch for green gc?

Reply:
O, it's not supported in tests.

Reply:
This PR is not related to other parts of Green GC and won't be able to switch off.